src/Resolution.test.ts

```diff
@@ -27,6 +27,10 @@ try {
   console.warn('dotenv is not installed');
 }
 
+// Live test are failing because default blockchain provider linkpool is very slow. 
+// Had to increase timeout for async operations from 5 to 12 seconds
+jest.setTimeout(12000);
```

Comment:
It is better to have a common setup file if you want it to be in every test. It is done via `jest.config.js` option: https://jestjs.io/docs/en/configuration#setupfiles-array